crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -843,7 +844,8 @@ impl<T: WasmInstance> WasmInstanceActor<T> {
             let mut index_for_column = None;
             for index in table.indexes.iter() {
                 let [index_col_id] = *index.col_ids else {
-                    anyhow::bail!("multi-column indexes not yet supported")
+                    //Ignore multi-column indexes
```

Comment:
Why are we ignoring multi-column indexes?

Reply:
This needs constraint to fully resolve this info, and that comes in the next PR.